packages/gestalt/src/ModuleExpandableItem.js

```diff
@@ -1,83 +1,88 @@
 // @flow strict
-import React, { Fragment, type Node } from 'react';
+import React, { type Node } from 'react';
 import Box from './Box.js';
+import Flex from './Flex.js';
 import Icon from './Icon.js';
+import ModuleTitle from './ModuleTitle.js';
 import TapArea from './TapArea.js';
 import Text from './Text.js';
-import { type BaseProps, type ExpandableItemProps } from './moduleTypes.js';
-import ModuleTitle from './ModuleTitle.js';
+import { type ModuleExpandableItemBaseProps } from './moduleTypes.js';
 
 type Props = {|
-  ...BaseProps,
-  ...ExpandableItemProps,
+  ...ModuleExpandableItemBaseProps,
+  accessibilityExpandLabel: string,
+  accessibilityCollapseLabel: string,
+  id: string,
+  isCollapsed: boolean,
+  onModuleClicked: (boolean) => void,
 |};
 
 export default function ModuleExpandableItem({
-  id,
-  title,
+  accessibilityCollapseLabel,
+  accessibilityExpandLabel,
+  children,
   icon,
   iconAccessibilityLabel,
-  accessibilityExpandLabel,
-  accessibilityCollapseLabel,
-  summary,
+  id,
   isCollapsed,
   onModuleClicked,
+  summary,
+  title,
   type = 'info',
-  children,
 }: Props): Node {
   return (
-    <Fragment>
-      <TapArea
-        onTap={() => {
-          onModuleClicked(!isCollapsed);
-        }}
-        accessibilityLabel={isCollapsed ? accessibilityExpandLabel : accessibilityCollapseLabel}
-        accessibilityControls={id}
-        accessibilityExpanded={!isCollapsed}
-      >
-        <Box padding={6} display="flex">
-          <Box display="flex" flex="grow" marginEnd={6} alignItems="baseline">
-            <Box column={isCollapsed && summary ? 6 : 12} display="flex">
-              <ModuleTitle
-                type={type}
-                title={title}
-                icon={icon}
-                iconAccessibilityLabel={iconAccessibilityLabel}
-              />
-            </Box>
-            {summary && isCollapsed && (
-              <Box column={6} marginStart={6}>
-                {summary.map((item, i) => (
-                  <Box key={i} marginBottom={i === summary.length - 1 ? 0 : 2}>
-                    <Text size="md" truncate>
-                      {item}
-                    </Text>
-                  </Box>
-                ))}
+    <Box padding={6}>
+      <Flex direction="column" gap={6}>
+        <TapArea
+          accessibilityControls={id}
+          accessibilityExpanded={!isCollapsed}
+          accessibilityLabel={isCollapsed ? accessibilityExpandLabel : accessibilityCollapseLabel}
+          onTap={() => {
+            onModuleClicked(!isCollapsed);
+          }}
+        >
+          <Flex>
+            <Box alignItems="baseline" display="flex" flex="grow" marginEnd={6}>
```

Comment:
You'd think this would be easily replaced with Flex by moving the `marginEnd` to `gap`on the parent, but it's not quite that simple. I didn't feel like spending the time to debug it now, since it doesn't really matter and this PR is already pretty bloated 😅